saltedge-compliance-connector-java-v2/src/main/java/com/saltedge/connector/example/controllers/consent/UserAuthenticateController.java

```diff
@@ -40,62 +40,55 @@ public class UserAuthenticateController extends ConsentBaseController {
   @GetMapping(BASE_PATH)
   public ModelAndView showSignIn(
       @RequestParam(name = SDKConstants.KEY_SCOPE) Scope scope,
-      @RequestParam(name = SDKConstants.KEY_SESSION_SECRET, required = false) String sessionSecret,
-      @RequestParam(name = SDKConstants.KEY_PAYMENT_ID, required = false) String paymentId
+      @RequestParam(name = SDKConstants.KEY_STATE, required = false) String state
+//      @RequestParam(name = SDKConstants.KEY_PAYMENT_ID, required = false) String paymentId
   ) {
     if (scope == null) throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "invalid scope");
-    if (!StringUtils.hasLength(sessionSecret) && !StringUtils.hasLength(paymentId)) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "no session_secret or payment_id");
+    if (!StringUtils.hasLength(state)) {
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "no state");
     }
 
-    ModelAndView result = createSignInModel(scope);
-    if (Scope.payments.equals(scope)) {
-      return result.addObject(SDKConstants.KEY_PAYMENT_ID, paymentId);
-    } else {
-      return result.addObject(SDKConstants.KEY_SESSION_SECRET, sessionSecret);
-    }
+    return createSignInModel(scope).addObject(SDKConstants.KEY_STATE, state);
   }
 
   // Receive SignIn page credentials
   @PostMapping(BASE_PATH)
   public ModelAndView onSubmitCredentials(
       @RequestParam(name = "submit") boolean positiveAction,
       @RequestParam(name = SDKConstants.KEY_SCOPE) Scope scope,
-      @RequestParam(name = SDKConstants.KEY_SESSION_SECRET, required = false) String sessionSecret,//AIS or PIIS session secret
-      @RequestParam(name = SDKConstants.KEY_PAYMENT_ID, required = false) String paymentId,//PIS
+      @RequestParam(name = SDKConstants.KEY_STATE, required = false) String state,//AIS, PIS, PIIS session secret
+//      @RequestParam(name = SDKConstants.KEY_PAYMENT_ID, required = false) String paymentId,//
```

Comment:
I see that PIS is mentioned in line 59, so I think we can delete this line.

Reply:
deleted